analyzer/src/main/scala/io/scalac/newspaper/analyzer/AnalyzerRunner.scala

```diff
@@ -5,12 +5,12 @@ import akka.kafka.ProducerMessage
 import akka.kafka.scaladsl.{Consumer, Producer}
 import akka.kafka.{Subscriptions, ConsumerSettings, ProducerSettings}
 import akka.stream.ActorMaterializer
-import akka.stream.scaladsl.Sink
```

Comment:
It's a leftover :/ I was testing if it does work, but publishing few messages here to see if they reach mailer - they do